"rainbow" ice nucleation parametrization from Frostenberg et al 2023 (#319)
CliMA/CloudMicrophysics.jl: docs/src/IceNucleation.md

  - the parameterization of activation of dust aerosol particles into ice crystals
    via deposition of water vapor,
  - water activity based parameterization of immersion freezing,
  - water activity based parameterization of homogeneous freezing,

>   - the parametrization of temperature-dependent ice nucleating particles concentration (including randomness),

Reviewer: ```suggestion
  - the parametrization of temperature-dependent aerosol-independent ice nucleating particles concentration
```

    The current parameterization in CloudMicrophysics.jl is not valid for \Delta a_w
    values that are obtained from pure water droplets. Though CliMA lines look far
    from the Spichtinger 2023 line, the lines seem to move closer as x approaches 0.

### INP Concentration Frequency

> With this parametrization, the concentration of ice nucleating particles (INPs) is found by randomly drawing from a relative frequency distribution, which depends on the temperature. It is based on [Frostenberg2023](@cite) and represents the randomness of INP concentrations. The scheme does not require information about aerosol. The relative frequency distribution ``D`` of INP concentration is derived from measurements, for marine

Reviewer: ```suggestion
With this parametrization, the concentration of ice nucleating particles (INPs) is found based on the relative frequency distribution, which depends on the temperature, but does not depend on aerosol types. It is based on [Frostenberg2023](@cite) and is derived from measurements, for marine
```

> data sets (as described in [Frostenberg2023](@cite)). It is a lognormal distribution, described by

Reviewer: ```suggestion
data sets. It is a lognormal distribution, described by
```

```math
\begin{equation}
  D(\mu, \sigma^2) = \frac{1}{\sqrt{2 \pi} \sigma} exp \left(- \frac{[ln(a \cdot INPC) - \mu(T)]^2}{2 \sigma^2} \right),
\end{equation}
```
where ``T`` is the temperature in degrees Celsius, ``INPC`` is the INP concentration in m``^{-3}`` and the temperature-dependent mean ``\mu(T)`` is given by
```math
\begin{equation}
  \mu(T) = ln(-(b \cdot T)^9 \times 10^{-9})
\end{equation}
```
 ``\sigma^2`` is the variance, ``a`` and ``b`` are coefficients. The parameters defined in [Frostenberg2023](@cite) for marine data sets are ``\sigma^2=1.37``, ``a=1 m^3``, ``b=1 [/ \degree C]``.

> 

Reviewer: The degree symbol doesn't render properly on the preview. Maybe deg will work. If not, I'm fine to just have `C`

Author: I'm not sure how to see the preview, but I hope it'll work now

Reviewer: No, it still doesn't render properly. Lets just do `1/C` and not spend time on this. 

Could I ask you to change the temperature in the plots to be in Kelvin?

 !!! note

> 

Reviewer: I think we can't have an empty line between note and note message

>     The function for INP conentration frequency in our source code takes ``T`` in Kelvin.

Reviewer: ```suggestion
   Our implementation uses base SI units and takes ``T`` in Kelvin.
```

```@example
include("plots/Frostenberg_fig1.jl")

> ```

Reviewer: Could you also include the `Frostenberg_fig1_T16.svg` plot?

    The current parameterization in CloudMicrophysics.jl is not valid for \Delta a_w
    values that are obtained from pure water droplets. Though CliMA lines look far
    from the Spichtinger 2023 line, the lines seem to move closer as x approaches 0.

> ### INP Concentration Frequency

Reviewer: ```suggestion
## INP Concentration Frequency
```
